projects/core/src/auth/user-auth/events/login-event.builder.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { StateEventService } from '../../../state/event/state-event.service';
+import { AuthActions } from '../store/actions';
+import { LoginEvent } from './user-auth.events';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LoginEventBuilder {
+  constructor(protected stateEventService: StateEventService) {
+    this.register();
+  }
+
+  /**
+   * Registers logout events
+   */
+  protected register(): void {
+    this.logoutEvent();
+  }
+
+  /**
+   * Register a logout event
+   */
+  protected logoutEvent(): void {
```

Comment:
The name of the builder is Login, while this method's name is logout 🤔😉

Reply:
Oops, copy pasta